### PR TITLE
audio subsystem automation

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**
Signed off by Przemysław Banasiak (przemyslaw.banasiak@3mdeb.com)

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md [hunk 1]
```diff
@@ -291,24 +301,23 @@ after plugging in the 3.5 mm jack into the slot.
...
+```
 
 **Expected result**
 
-1. `Jack Information` field in the first case should show the position
-    `Front Panel 3.5 mm Jack`.
-1. After disconnecting a headset from the laptop and checking again
-    option `Addtional device properties` field `Jack Information` should not
-    contain the phrase `Front Panel 3.5 mm Jack`.
+1. External headset should be listed in the output.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md:320]: In think, that in section `Expected result` we should add output example

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md [hunk 2]
```diff
@@ -280,35 +298,35 @@ after plugging in the 3.5 mm jack into the slot.
 
 **Test setup**
 
-1. Proceed with the [Common](#common) section.
+1. Proceed with the Common section.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md:301]: Why this hyperlink has been erased?

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md (hunk 2 above)
Comment (PrzemyslawBanasiak13) [reply docs/unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md:301]: Deleted accidentally

Comment (PrzemyslawBanasiak13) [inline docs/unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md:301]:
```suggestion
1. Proceed with the [Common](#common) section.
```

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**